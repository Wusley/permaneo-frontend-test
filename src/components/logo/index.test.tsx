import { render } from '@testing-library/react'
import Logo from './'

jest.mock( './link', () => ( {
  link: {
    icon: jest.fn( () => <svg data-testid="logo-icon" /> ),
    text: 'Test Logo'
  }
} ) )

describe( 'Logo Component', () => {
  it( 'matches the snapshot', () => {
    const { container } = render( <Logo /> )
    expect( container ).toMatchSnapshot()
  } )
} )