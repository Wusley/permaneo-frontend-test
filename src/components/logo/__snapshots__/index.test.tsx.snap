// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component matches the snapshot 1`] = `
<div>
  <h1
    class="chakra-heading css-1xrx6ul"
  >
    <div
      class="css-rjlbth"
    >
      <svg
        data-testid="logo-icon"
      />
      Test Logo
    </div>
  </h1>
</div>
`;
